world_frame: "map"
anchor_tag: "tag36h11:0"
tags:
  tag36h11:0:
    position: {x: -2.000000, y: 3.250000, z: 0.500000}
    orientation: {x: 0.500000, y: 0.500000, z: -0.500000, w: 0.500000}
  tag36h11:1:
    position: {x: 0.000000, y: 3.250000, z: 0.500000}
    orientation: {x: 0.500000, y: 0.500000, z: -0.500000, w: 0.500000}
  tag36h11:2:
    position: {x: 2.000000, y: 3.250000, z: 0.500000}
    orientation: {x: 0.500000, y: 0.500000, z: -0.500000, w: 0.500000}
  tag36h11:3:
    position: {x: -2.000000, y: -3.250000, z: 0.500000}
    orientation: {x: -0.500000, y: 0.500000, z: 0.500000, w: 0.500000}
  tag36h11:4:
    position: {x: 0.000000, y: -3.250000, z: 0.500000}
    orientation: {x: -0.500000, y: 0.500000, z: 0.500000, w: 0.500000}
  tag36h11:5:
    position: {x: 2.000000, y: -3.250000, z: 0.500000}
    orientation: {x: -0.500000, y: 0.500000, z: 0.500000, w: 0.500000}
  tag36h11:6:
    position: {x: -3.000000, y: 0.600000, z: 0.500000}
    orientation: {x: 0.707108080, y: -0.000001299, z: -0.707105482, w: -0.000001299}
  tag36h11:7:
    position: {x: -3.000000, y: -0.600000, z: 0.500000}
    orientation: {x: 0.707108080, y: -0.000001299, z: -0.707105482, w: -0.000001299}
  tag36h11:8:
    position: {x: -1.000000, y: 0.600000, z: 0.500000}
    orientation: {x: 0.707108080, y: -0.000001299, z: -0.707105482, w: -0.000001299}
  tag36h11:9:
    position: {x: -1.000000, y: -0.600000, z: 0.500000}
    orientation: {x: 0.707108080, y: -0.000001299, z: -0.707105482, w: -0.000001299}
  tag36h11:10:
    position: {x: 1.000000, y: 0.600000, z: 0.500000}
    orientation: {x: 0.707108080, y: -0.000001299, z: -0.707105482, w: -0.000001299}
  tag36h11:11:
    position: {x: 1.000000, y: -0.600000, z: 0.500000}
    orientation: {x: 0.707108080, y: -0.000001299, z: -0.707105482, w: -0.000001299}
  tag36h11:12:
    position: {x: 3.300000, y: 0.600000, z: 0.500000}
    orientation: {x: 0.707108080, y: -0.000001299, z: -0.707105482, w: -0.000001299}
  tag36h11:13:
    position: {x: 3.300000, y: -0.600000, z: 0.500000}
    orientation: {x: 0.707108080, y: -0.000001299, z: -0.707105482, w: -0.000001299}
  tag36h11:14:
    position: {x: 3.300000, y: 2.520000, z: 0.500000}
    orientation: {x: -0.653281, y: -0.270598, z: 0.653281, w: -0.270598}
  tag36h11:15:
    position: {x: 3.300000, y: -2.520000, z: 0.500000}
    orientation: {x: -0.653281, y: 0.270598, z: 0.653281, w: 0.270598}
  tag36h11:16:
    position: {x: -3.000000, y: 2.520000, z: 0.500000}
    orientation: {x: 0.270598, y: 0.653281, z: -0.270598, w: 0.653281}
  tag36h11:17:
    position: {x: -3.000000, y: -2.520000, z: 0.500000}
    orientation: {x: -0.270598, y: 0.653281, z: 0.270598, w: 0.653281}
  tag36h11:18:
    position: {x: -1.100000, y: 3.250000, z: 0.500000}
    orientation: {x: 0.500000, y: 0.500000, z: -0.500000, w: 0.500000}
  tag36h11:19:
    position: {x: 0.900000, y: 3.250000, z: 0.500000}
    orientation: {x: 0.500000, y: 0.500000, z: -0.500000, w: 0.500000}
  tag36h11:20:
    position: {x: 2.900000, y: 3.250000, z: 0.500000}
    orientation: {x: 0.500000, y: 0.500000, z: -0.500000, w: 0.500000}
  tag36h11:21:
    position: {x: -1.100000, y: -3.250000, z: 0.500000}
    orientation: {x: -0.500000, y: 0.500000, z: 0.500000, w: 0.500000}
  tag36h11:22:
    position: {x: 0.900000, y: -3.250000, z: 0.500000}
    orientation: {x: -0.500000, y: 0.500000, z: 0.500000, w: 0.500000}
  tag36h11:23:
    position: {x: 2.900000, y: -3.250000, z: 0.500000}
    orientation: {x: -0.500000, y: 0.500000, z: 0.500000, w: 0.500000}
  tag36h11:24:
    position: {x: -2.980000, y: 0.600000, z: 0.500000}
    orientation: {x: 0.500000, y: 0.500002, z: 0.500000, w: 0.499998}
  tag36h11:25:
    position: {x: -2.980000, y: -0.600000, z: 0.500000}
    orientation: {x: 0.500000, y: 0.500002, z: 0.500000, w: 0.499998}
  tag36h11:26:
    position: {x: -0.980000, y: 0.600000, z: 0.500000}
    orientation: {x: 0.500000, y: 0.500002, z: 0.500000, w: 0.499998}
  tag36h11:27:
    position: {x: -0.980000, y: -0.600000, z: 0.500000}
    orientation: {x: 0.500000, y: 0.500002, z: 0.500000, w: 0.499998}
  tag36h11:28:
    position: {x: 1.020000, y: 0.600000, z: 0.500000}
    orientation: {x: 0.500000, y: 0.500002, z: 0.500000, w: 0.499998}
  tag36h11:29:
    position: {x: 1.020000, y: -0.600000, z: 0.500000}
    orientation: {x: 0.500000, y: 0.500002, z: 0.500000, w: 0.499998}
  tag36h11:30:
    position: {x: 3.320000, y: 0.600000, z: 0.500000}
    orientation: {x: 0.500000, y: 0.500002, z: 0.500000, w: 0.499998}
  tag36h11:31:
    position: {x: 3.320000, y: -0.600000, z: 0.500000}
    orientation: {x: 0.500000, y: 0.500002, z: 0.500000, w: 0.499998}
